Sources/StreamChat/WebSocketClient/Events/NotificationEvents.swift

```diff
@@ -4,44 +4,44 @@
 
 import Foundation
 
-public struct NotificationMessageNewEvent<ExtraData: ExtraDataTypes>: EventWithMessagePayload, EventWithChannelId {
+public struct NotificationMessageNewEvent: EventWithMessagePayload, EventWithChannelId {
     public let userId: UserId
     public let cid: ChannelId
     public let messageId: MessageId
     public let createdAt: Date
-    public let unreadCount: UnreadCount
+    public let unreadCount: UnreadCount?
     let payload: Any
     
-    init(from response: EventPayload<ExtraData>) throws {
+    init<ExtraData: ExtraDataTypes>(from response: EventPayload<ExtraData>) throws {
         userId = try response.value(at: \.message?.user.id)
         cid = try response.value(at: \.channel?.cid)
         messageId = try response.value(at: \.message?.id)
         createdAt = try response.value(at: \.message?.createdAt)
-        unreadCount = try response.value(at: \.unreadCount)
+        unreadCount = try? response.value(at: \.unreadCount)
```

Comment:
In which cases backend doesn't send this value? would be nice to comment this

Reply:
There are several conditions. See https://getstream.slack.com/archives/CE5N802GP/p1617740759146000 for more info. This whole thing about unread counts is still not ideal and will need a rethink.